Tests/conf.json

```diff
@@ -85,6 +85,12 @@
                 "Mail_Sender_(New)_STARTTLS"
             ]
         },
+        {
+            "integrations":"Azure Network Security Groups",
+            "fromversion": "6.0.0",
```

Comment:
why is it fromversion 6.0 ?

Reply:
On accident. Changed it everywhere but here. Changing back now